src/ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -857,11 +857,25 @@ def _launch(self, start_parm, timeout=10):
             raise MapdlRuntimeError(
                 "Can only launch the GUI with a local instance of MAPDL"
             )
-        from ansys.mapdl.core.launcher import launch_grpc
+        from ansys.mapdl.core.launcher import generate_mapdl_launch_command, launch_grpc
 
         self._exited = False  # reset exit state
-        port, directory, process = launch_grpc(**start_parm)
-        self._connect(port)
+
+        args = self._start_parm
+        cmd = generate_mapdl_launch_command(
+            exec_file=args["exec_file"],
+            jobname=args["jobname"],
+            nproc=args["nproc"],
+            ram=args["ram"],
+            port=args["port"],
+            additional_switches=args["additional_switches"],
+        )
+
+        process = launch_grpc(
+            cmd=cmd, run_location=args["run_location"], env_vars=self._env_vars or None
```

Comment:
the passing of `_env_vars` looks like its new behavior, since it wasn't part of `start_parm`

Reply:
The idea behind this PR is to "hide" or simplify `launch_grpc`.... and always use `launch_mapdl` ... 

instead of using `launch_grpc`, we will use `launch_grpc(mode='grpc')` ... if you just want to launch the instance,
you can pass the `just_launch` argument (I might rename it to `only_launch`).

Reply:
Also, ... before `launch_grpc` was accepting the following arguments:
```py
     add_env_vars: Optional[Dict[str, str]] = None,
     replace_env_vars: Optional[Dict[str, str]] = None,
```
hence, inside it was doing some env vars manipulation. I have decided to take that off... now all the env vars manipulation is done in `launch_grpc`.

I wanted to reduce code duplication, that is why I took so much stuff out of `launch_grpc`. Much of that stuff was already done inside `launch_grpc`.